lib/ui_kit/organisms/cards/ui_kit_find_someone_card.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:flutter/material.dart';
+import 'package:shuffle_uikit/shuffle_uikit.dart';
+
+class UiKitFindSomeoneCard extends StatelessWidget {
+  final String? avatarUrl;
+  final String? userNickName;
+  final String? userName;
+  final int? userPoints;
+  final int? sameInterests;
+  final VoidCallback? onMessage;
+
+  const UiKitFindSomeoneCard({
+    super.key,
+    this.avatarUrl,
+    this.userNickName,
+    this.userName,
+    this.userPoints,
+    this.sameInterests,
+    this.onMessage,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    double calculatedWidth = MediaQuery.of(context).size.width;
```

Comment:
not like this. this reduce perfomance of the layout. USE 1.sw (flutter_screenutil package)